tests/python/test_unit.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import smt_switch as ss
+
+from collect_solvers import solvers, full_bv_support
+
+@pytest.mark.parametrize("create_solver", full_bv_support)
+def test_unit_op(create_solver):
```

Comment:
Python tests pass when I run them, but I think there needs to be more coverage.

Reply:
Agreed. I'll add more tests soon.

Reply:
Added a few more tests in 4bdf66fb7f6f76012d0a2f7ef57f075e9954a864, 94bf63f33c1e4306d5ffcb409b0a9333e0544c69, and b0d1fb35131817168f1060a9bfce4b59693a355e